### PR TITLE
Chore merge from aa50e23 to 92f9dda

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/aa50e2361f6d51137a195907469338a14938147b
- [x] https://github.com/twbs/bootstrap/commit/33ad33843a4235804dd98553a7734942c1ba629f
- [x] https://github.com/twbs/bootstrap/commit/3b9b154563b254cdc774ab74288658e0fc440014
- [x] https://github.com/twbs/bootstrap/commit/208ba3d7488495f7ea1129d0ebaf1576bc58a272
- [x] https://github.com/twbs/bootstrap/commit/d533e6f33de2a26a0b7bf5a577eb5c120e51face
  - ~~Based on the recent work on tooltips and popovers, I've only included modifications from Bootstrap as "Boosted mod"s. @MewenLeHo could you please check this commit and see if things can be modified or if I need to keep them as "Boosted mod"s?~~
  - Integrated via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1927/commits/686532a7591d40a62d61fb034ea44aeb0865ec04
- ~~https://github.com/twbs/bootstrap/commit/f77117141dfe34d8fe984eabddf660e29cb96a57~~: nothing to do here since we don't have any dark carousels
- [x] https://github.com/twbs/bootstrap/commit/cd890881cb1bf54375053e8d20015dcb896ae85f
- [x] https://github.com/twbs/bootstrap/commit/74431bdb933cd7cd5599bf4dde74234a13ed5902
- [x] https://github.com/twbs/bootstrap/commit/43bc20a59eb2b8d65a12296c677ae331b85b3c58
- ~~https://github.com/twbs/bootstrap/commit/d76ee2846f420768a8a3fb70a0eefc37df2d951e~~: nothing to do here since we don't have specific migration guide between alpha versions
- [x] https://github.com/twbs/bootstrap/commit/2bbf670fc2b230f18ba763f50498b8ee90f751c3
- [x] https://github.com/twbs/bootstrap/commit/1729bcca1b3a842829c9a64fe1f34fdd53465189
- [x] https://github.com/twbs/bootstrap/commit/e34cf6a34ac9e81e63df01daa57c38b5c3712dee
- [x] https://github.com/twbs/bootstrap/commit/31f93e2aff5050877f499faca05dc4341942a40a
- [x] https://github.com/twbs/bootstrap/commit/06f7c3b6c73580cd05b131bdab4d156a864d418c
- [x] https://github.com/twbs/bootstrap/commit/94c7dc717ee03749198445bd4d69b3597d5a1d2a
- [x] https://github.com/twbs/bootstrap/commit/cad38efcaee09c209b13842dbccbdb4a4c933852
- [x] https://github.com/twbs/bootstrap/commit/92f9dda263e4a80a8b9a60574af20f610b6c6fa8
  - ⚠️ @louismaximepiton Please double-check in details these modifications for non-regression

### CODE REVIEW

**Comment by MewenLeHo** [comment]
>   * Based on the recent work on tooltips and popovers, I've only included modifications from Bootstrap as "Boosted mod"s. @MewenLeHo could you please check this commit and see if things can be modified or if I need to keep them as "Boosted mod"s?

For me we can't include their modifications if we want to keep what was merged yestersday.

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
I think we can easily implement the Bs changes about the tooltips by changing the offset in the Js to `[0, 10]`. At least I couldn't spot any big breaking change in here.

Some changes are missing anyway for me at least.

File: scss/_variables.scss [hunk 1]
```diff
@@ -1900,7 +1903,7 @@ $alert-padding-y:                   1rem !default;
 $alert-padding-x:                   $spacer !default;
 $alert-margin-bottom:               $spacer !default;
 $alert-color:                       inherit !default; // Boosted mod
-$alert-border-radius:               $border-radius !default;
+$alert-border-radius:               var(--#{$prefix}border-radius) !default;
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply scss/_variables.scss:1906]: Oh good catch, forgot about this one (should be the only forgotten one). Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1927/commits/bbf99c066d0ee6e3c8f85e377b509190d913eeed

File: site/assets/js/snippets.js [hunk 2]
```diff
@@ -69,28 +70,43 @@
...
   // -------------------------------
-  // Used in 'Show live toast' example in docs or StackBlitz
-  const alertPlaceholder = document.getElementById('liveAlertPlaceholder')
-  const alertTrigger = document.getElementById('liveAlertBtn')
+  // Used in 'Show live alert' example in docs or StackBlitz
 
-  // Boosted mod: adapted innerHTML to have the icon and so added a parameter within alert()
+  // Boosted mod: adapted innerHTML to have the icon and so added a parameter within `alert()` function
+  // js-docs-start live-alert
+  const alertPlaceholder = document.getElementById('liveAlertPlaceholder')
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/assets/js/snippets.js]: Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1927/commits/a12d3302e7f70f350436c32e0700a6dfd54b4cbd

**Comment by julien-deramond** [comment]
Based on the latest comment by @louismaximepiton, I tried to gather the modification and only change the `offset` to `[0, 10]`. Just checked quickly and seems to do the job. @MewenLeHo could you check it precisely since you know well the topic; the corresponding commit is https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1927/commits/686532a7591d40a62d61fb034ea44aeb0865ec04.

**Review by louismaximepiton — APPROVED** [review]

**Comment by julien-deramond** [comment]
Put you as a reviewer @MewenLeHo just of this last part

**Review by MewenLeHo — APPROVED** [review]
Last part seems good.

**Pull request merged**